docker/admin/entrypoint.sh

```diff
@@ -24,6 +29,9 @@ if [[ $GITHUB_ACCOUNT_SSH_USERS ]]; then
 	IFS=$ORIGINAL_IFS
 fi
 
+# export the ENV vars globally so they can be available for ssh users at login
+printenv | grep -E 'ILIOS_|TRUSTED_PROXIES|DSN' | sed 's/^/export /g' > /etc/profile.d/ecs_env.sh
```

Comment:
I'm surprised this is needed, testing locally. Wondering if this is the best way to push this forward. mainly I'm worried we miss something. Like `APP_ENV` (which isn't covered here).